pmarks-net/ipvfoo
# Add Firefox Support

Firefox now supports WebExtensions, more or less the same as Chrome and Opera. This seems to work with no significant changes.

Also added a makefile for building the xpi, but I don't know how to package for chrome.

To test:
Clone and run `make`
In Firefox, go to `about:debugging`, click "Load temporary add-on", select xpi.
Reload a page and the indicator should appear.
